docker_etl/verify_files.py

```diff
@@ -0,0 +1,33 @@
+import os
+from typing import List, Set, Tuple
+
+from docker_etl.file_utils import CI_JOB_NAME, CI_WORKFLOW_NAME, get_job_dirs
+
+REQUIRED_FILES = {CI_JOB_NAME, CI_WORKFLOW_NAME, "README.md", "Dockerfile"}
+
+
+def check_missing_files() -> List[Tuple[str, Set[str]]]:
```

Comment:
I think the yaml files should be checked that they are well formed in this file. It might be as simple as `import yaml; yaml.read(open(CI_JOB_NAME))` and checking that it doesn't except. It could also be using `yamllint` in the wrapper script.

Reply:
Added yaml read as part of the ci config update